bloom-housing/bloom
# 1352/Add flagged and mark as duplicate to CSV export

# Pull Request Template

## Issue #1352 

Addresses # (issue)

## Description

Adds flagged and marked as duplicate to CSV export.

## Type of change


## How Can This Be Tested/Reviewed?

Log into partners and export CSV for a listing that have both flagged and duplicated applications. Downloaded file should contain flagged and marked as duplicate columns at the end.


## Checklist:
